KS-4-5 задача 2

VVShchegolev opened the pull request.

Commit:
KS-4-5 задача 2

ievetrovdev commented on src/main/kotlin/Lesson_4_Ex_2/Lesson_4_Ex_2.kt:
```diff
@@ -0,0 +1,18 @@
+    val volume2 = 100
+
+    println("Груз с весом $weight1 кг и объемом $volume1 л соответствует категории 'Average': ${weight1 > MIN_WEIGHT_KG && weight1 <= MAX_WEIGHT_KG && volume1 < MAX_VOLUME_LITERS}")
+    println("Груз с весом $weight2 кг и объемом $volume2 л соответствует категории 'Average': ${weight2 > MIN_WEIGHT_KG && weight2 <= MAX_WEIGHT_KG && volume2 < MAX_VOLUME_LITERS}")
```
Да, код корректный. Но давай чуть улучшим.
Строки слишком длинные (>120 символов), глазу труднее в них ориентироваться. Часто в компаниях вводят правила форматирования, а также специальные утилиты, которые следят за соблюдением правил.

Вот например такая утилита Detekt для Kotlin запрещает по-умолчанию строки длиннее 120 символов: https://detekt.dev/docs/rules/formatting/#maximumlinelength.
Давай добавим перенос или вынесем в переменную условие

ievetrovdev reviewed (changes requested):
Обрати внимание на идентификатор задачи в названии пулл реквеста. Это задача `KS-4-2`, а не `KS-4-5`

VVShchegolev committed:
Update Lesson_4_Ex_2.kt

Исправленные недочеты:
Добавил переменные "isAverage1" и "isAverage2" для условия, чтобы код можно было легче читать

ievetrovdev reviewed (approved):
Отлично!
